### ISSUE TITLE
Alert dialog hangs test execution if it is opened from a frameset

### ISSUE THREAD

**Issue by umaar** [DevExpress/testcafe#1210]
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

Alert box error

### What is the expected behavior?

No alert box error

Hey! Really sorry for such a vague and incomplete test report but checkout this screenshot:

![screen shot 2017-02-06 at 12 22 34 pm](https://cloud.githubusercontent.com/assets/381495/22647368/f9bf0d4c-ec68-11e6-9a98-1fedbd1079d0.png)

I don't suppose you have any idea what could cause that? I'm not even sure if it's from the webpage or it's a Testcafe thing. Feel free to close this issue if you don't know, I appreciate you need more information to debug this one.

**Comment by georgiy-abbasov** [comment]
Hello Umar!
Thanks for report ! We need some additional information about reproducing this issue. Please, post here the test code and the command which launches TestCafe (we need to know OS X and Chrome versions). You can omit your credentials.

**Comment by umaar** [comment]
Test command:

```
SAUCE_USERNAME=xxx SAUCE_ACCESS_KEY=xxx ./node_modules/testcafe/bin/testcafe.js --skip-js-errors 'saucelabs:Chrome@55.0:OS X 10.10' src/test/*.js
```

Testcafe plugin versions:

```
"testcafe": "0.13.0-alpha1"
"testcafe-browser-provider-saucelabs": "^1.1.0"
```

Chrome Version:

```
Chrome@55.0:OS X 10.10
```

As for the test code itself, I believe it's this:

```
test.page('http://genomemedicine.biomedcentral.com/')('Submit Manuscript button', async t => {
    await t.click('.journal-identity__submit');
    await t.wait(1000);
    const expectedUrl = 'http://www.editorialmanager.com/gmed/default.aspx';
    const currentUrl = await utils.pageUrl();
    await t.expect(currentUrl).eql(expectedUrl);
});

// pageUrl() is implemented as follows

const pageUrl = clientFunction(() => window.location.href);
```

**Comment by umaar** [comment]
I should add that I can't reproduce it right now. But it did happen a few times.

**Comment by georgiy-abbasov** [comment]
Thank you for explanation! We have reproduced the issue and currently working on it.

**Comment by georgiy-abbasov** [comment]
@umaar, i'm sorry for late response, afaik it't doesnt relate to SauceLabs running, because i've reproduced it on local machine. I've created separate issue for investigating this problem - https://github.com/DevExpress/testcafe/issues/1336
![Screenshot](https://content.screencast.com/users/GeorgeZA/folders/Jing/media/4cee9059-8ef4-433c-b1a4-d85963f6cb75/2017-03-21_1458.png)

**Comment by AndreyBelym** [comment]
Hello @umaar! We discovered that we have problems with `<frameset>`, you can track the issue here: https://github.com/DevExpress/testcafe-hammerhead/issues/1070

**Comment by LavrovArtem** [comment]
fixed in `testcafe-hammerhead@10.8.1`
